lib/printer.js

```diff
@@ -1486,6 +1502,9 @@ function printFunctionParams(path, options, print) {
     if (joined.length > 1 ||
         joined.getLineLength(1) > options.wrapColumn) {
         joined = fromString(",\n").join(printed);
+        if (options.trailingComma && !fun.rest) {
```

Comment:
Ah, yes, I remember a discussion at TC39 about this: you're never going to add another parameter after a `...rest`, so the argument from minimizing diff lines does not apply.